src/helpers/bot_stats_poster.ts

```diff
@@ -1,4 +1,4 @@
-import { Client } from "discord.js";
```

Comment:
Couldn't you make changing this file less work by just importing client from eris?

Reply:
Imports in node are cached, so every file importing will receive the same reference. We do `import * as Eris` in a bunch of a files, so doing it here wouldn't affect anything